examples/utils/example_client.rs

```diff
@@ -49,7 +49,7 @@ impl ExampleClient {
         let sign_keys = crypto::sign::gen_keypair();
         let encrypt_keys = crypto::box_::gen_keypair();
         let full_id = FullId::with_keys(encrypt_keys.clone(), sign_keys.clone());
-        let mut routing_client;
+        let routing_client = unwrap!(Client::new(sender, Some(full_id)));
```

Comment:
This line should also be left as it is - on master, `routing_client` is initialised in the loop below.